sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveSessionCatalog.scala

```diff
@@ -369,6 +369,13 @@ class ResolveSessionCatalog(
       AlterTableRecoverPartitionsCommand(
         v1TableName.asTableIdentifier,
         "ALTER TABLE RECOVER PARTITIONS")
+
+    case AlterTableChangeColumnStatement(tableName, columnName, newColumn) =>
+      val v1TableName = parseV1Table(tableName, "ALTER TABLE CHANGE COLUMN")
```

Comment:
Seems this command can also work on V2, if old column name and new column name is the same. cc @cloud-fan